include/gatekeeper_gt.h

```diff
@@ -21,15 +21,37 @@
 
 #include <stdint.h>
 
+#include <rte_ip.h>
+#include <rte_tcp.h>
+#include <rte_udp.h>
 #include <rte_atomic.h>
 
+#include "gatekeeper_config.h"
+
+struct gt_packet_fields {
+	uint16_t outer_ip_ver;
+	uint16_t inner_ip_ver;
+	uint8_t l4_proto;
+
+	union {
+		uint32_t v4;
+		uint8_t  v6[16];
+	} gatekeeper_server_ip;
+
+	void *l3_hdr;
+	void *l4_hdr;
+};
```

Comment:
`struct gt_packet_fields` already holds a number of pointers to headers, and with the change suggested in another comment, it'll only hold pointers and header type fields. Therefore the name `gt_packet_fields` doesn't properly describe this struct. Name suggestion: `gt_packet_headers`.